Modules/Microsoft365DSC/Modules/M365DSCReport.psm1

```diff
@@ -22,30 +18,16 @@ function New-M365DSCConfigurationToHTML
         [System.String]
         $OutputPath,
 
+        [Parameter()]
+        [System.String]
+        $TemplateName,
+
         [Parameter()]
         [Switch]
         $SortProperties
     )
 
-    if ([System.String]::IsNullOrEmpty($ParsedContent))
```

Comment:
Why are we trying to remove this? This is required for other scenarios that feed in the content for the report instead of a file. For example, when running from a serverless Azure function

Reply:
See comment below.